src/bin/pgcopydb/dump_restore.c

```diff
@@ -281,7 +281,13 @@ copydb_copy_database_properties_hook(void *ctx, SourceProperty *property)
 	PGSQL *dst = context->dst;
 	PGconn *conn = dst->connection;
 
-	const char *t_dbname = specs->connStrings.safeTargetPGURI.uriParams.dbname;
+	const char *t_pguri = specs->connStrings.safeTargetPGURI.uriParams.dbname;
+	const char *t_dbname = pgsql_escape_identifier(dst, t_pguri);
+
+	if (t_dbname == NULL)
+	{
+		return false;
+	}
```

Comment:
I think it's worth adding a log message about the situation actually. Because we don't expect it to ever happen, something like we have in similar unexpected places:

```
    log_error("BUG: failed to escape identifier \"%s\"", t_pguri);
```

Oh and also we went to fast with the naming of the intermediate variable. That's not a Postgres connection string (pguri), it's a database name... Please review naming!